### DISCUSSION TITLE
How to change table components and have them show in the browser?

### DISCUSSION THREAD

**Discussion by ethanpatchell** [rappasoft/laravel-livewire-tables#2095] [Q&A]
Hey all,

I am wanting to surround the tools bar and pagination bar in a <div class="card-body">.

However, I am unable to figure out how to get it to show on the website. 

Below is the config file.
`'theme' => 'bootstrap-5',

    'cache_assets' => true,
    /**
     * Enable or Disable automatic injection of assets
     */
    'inject_assets' => false,

    /**
     * Enable or Disable inclusion of published third-party assets
     */
    'published_third_party_assets' => false,

    /**
     * Enable or Disable remote third-party assets
     */
    'remote_third_party_assets' => true,`

I've also changed the "datatable.blade.php" page to this
`<div class="card-body">
        <x-livewire-tables::tools>
            <x-livewire-tables::tools.sorting-pills />
            <x-livewire-tables::tools.filter-pills />
            <x-livewire-tables::tools.toolbar />
        </x-livewire-tables::tools>
    </div>`
AND
`<div class="card-body">
        <x-livewire-tables::pagination :rows="$rows" />
    </div>`

I appreciate your help in advance :)

**Comment by lrljoe** [comment]
Editing the core views in this way is not supported nor recommend

**Reply by ethanpatchell** [reply]
Thanks for that. Is there a way I can add .card-body around the tools bar and pagination?

Am hoping to have the table full width but it's funny looking with the tools and pagination going right to the edge. I've attached a photo of how it currently is, and one of how I want it - from editing in dev tools.
![image](https://github.com/user-attachments/assets/11959400-da56-4d87-9693-150cb3e3ad7d)


![image](https://github.com/user-attachments/assets/3ed0bf40-b444-4053-8f1c-69096883bbab)

Thanks for the help!

**Reply by lrljoe** [reply]
Off the top of my head, I think there's a setToolsAttributes and setToolBarAttributes 

It'll be in the docs

That should let you add the classes at the relevant points

For the pagination, I think it's called setPaginationWrapperAttributes

Again it should be in the docs

Let me know if that doesn't work out!
